src/experiment.js

```diff
@@ -3,31 +3,49 @@ const path = require('path');
 const { FILEDIR } = require('./dirs');
 const { authMiddleware } = require('./authentication');
 
+const { timeStamp } = require('./serverUtils');
+
+var Config = require('./models/Config.model');
+
+const collection = 'experiment_data'
+
 module.exports = (router = new Router()) => {
-// Get config file
+  //get all config documents from mongo.
+  router.get('/', authMiddleware);
   router.get('/', (req, res) => {
-    res.sendFile(path.join(FILEDIR, 'config.txt'), {}, (err) => {
-      if (err) {
-        res.status(400).send({
-          message: 'no config',
-        });
-      }
+    Config.find({}).then(function(configs) {
+      var response = {};
+      var count = 1;
+      configs.forEach(function(record) { 
+        response[count] = JSON.parse(record["configData"]);
+        count += 1;
+      });
+      return Promise.all([response]);
+    }).then(function(respArray) {
+      res.status(200).set({
+      'Content-Type': 'application/json',
+      });
+      res.send(respArray[0]);
+    }).catch(function(error) {
+      res.status(500).send(error);
     });
   });
   // Post config requires password
   router.post('/', authMiddleware);
-  // Post config file
```

Comment:
Can we also get a route that updates an existing config? Right now we would just keep creating more and more configs indefinitely. 
Ex: sending a config via `PUT /config/2` would overwrite the 2nd config.

POST for creating new configs, PUT for updating old ones